invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html

```diff
@@ -90,11 +95,25 @@
 {%- endmacro %}
 
 
-{% macro file_list_box(files, pid, is_preview) %}
+{% macro file_list_box(files, pid, is_preview, record_access, format_date) %}
+  {% if record_access.files == 'restricted' and  record_access.embargo.active%}
+    {% set className = "embargoed" %}
+    {%- set panel_title = _("Restricted files") %}
+    {%- set popup_text = _("The files are currently under embargo and will be made publicly accessibly on ") + format_date(record_access.embargo.until) + "." %}
+  {% elif  record_access.files == 'restricted' %}
+    {% set className = "restricted" %}
+    {%- set panel_title = _("Restricted files") %}
+    {%- set popup_text = _("The files in this record are restricted to users with access.") %}
+  {% else %}
+    {%- set panel_title = _("Files") %}
+  {% endif %}
 <div class="panel-spacing">
-  <div class="ui accordion panel" id="preview" href="#collapsablePreview">
-    <div class="title panel-heading">
-      {{ _("Files") }}
+  <div class="ui accordion panel {{ className }}" id="preview" href="#collapsablePreview">
+    <div class="title panel-heading {{ className }}">
+      {{ panel_title }}
+      <span class="ui tooltip-popup text-muted"
+                          data-content="{{ popup_text }}"
+                        ><i class="question circle outline icon "></i></span>
```

Comment:
minor (give or take html whitespace issues)

```suggestion
      <span class="ui tooltip-popup text-muted" data-content="{{ popup_text }}">
        <i class="question circle outline icon "></i>
      </span>
```

Reply:
Do you guys use any formatter to avoid the formatting issues next time?